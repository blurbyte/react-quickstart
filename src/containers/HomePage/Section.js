import styled from 'styled-components';

const Section = styled.section`
  border-top: 1px solid #908e89;
  margin-top: 4rem;
`;

export default Section;
