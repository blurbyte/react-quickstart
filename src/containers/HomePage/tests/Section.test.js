import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Section from '../Section';

describe('<AppWrapper />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(<Section />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
