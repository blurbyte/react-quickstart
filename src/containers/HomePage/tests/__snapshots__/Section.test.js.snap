// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWrapper /> should render correctly 1`] = `
.ikogIq {
  border-top: 1px solid #908e89;
  margin-top: 4rem;
}

<section
  className="ikogIq"
/>
`;
